blog/2025/2025-05-09-blender-baking/index.md

```diff
@@ -0,0 +1,222 @@
+---
+title: 'The one true guide to baking materials in Blender'
+slug: 'blender-baking'
+description: 'How to get nice materials on low poly objects'
+date: '2025-05-09'
+authors: ['jaked']
+tags: ['blender', 'baking', 'normal-maps', 'article', 'tutorial', 'showcase']
+---  
+  
+Real time rendering performance is often limited by 3D assets as much as it's limited by code. Good low poly assets inevitably rely on baking, the process of transferring details from a high poly mesh with a complex material to a low poly one with a much simpler material. Unfortunately however there seems to be a lack of info regarding baking around, especially in Blender things can sometimes be a bit intuitive regarding baking. In the process of working on my game, A Short Odyssey, I came up with a workflow that works quite well for me, so I will share it with you today.
+
+For this tutorial we are going to use this wooden bowl model from the fantastic website [Polyhaven](https://polyhaven.com/a/wooden_bowl_02)
+
+![Bowl Rendered](bowl-rendered.webp)
+
+<!-- truncate -->
+
+As with all of the free CC0 models on Polyhaven this mesh has a fairly high number of small triangles, 4,666 to be exact, while that may not seem like a lot think about how big it is likely to be in a realtime scene. Most of the time the entire bowl might only be a few pixels tall! Especially given that small triangles are much more expensive than large triangles (due to quad occupancy) this is probably something we should deal with.
```

Comment:
As with all of the free CC0 models on Polyhaven, this mesh has a fairly high number of small triangles. 4,666 to be exact. While that may not seem like a lot...

Reply:
...than large triangles (due to quad occupancy). This is probably something...